js_functions/task2.js

```diff
@@ -0,0 +1,16 @@
+function checkAdult(age) {
```

Comment:
checkAdult is better name then it was before. But I can't figure out this function returns boolean value according to this name